tests/Resources/ti.network.httpclient.addontest.js

```diff
@@ -10,27 +10,35 @@
 var should = require('./utilities/assertions');
 
 describe('Titanium.Network.HTTPClient', function () {
+	this.timeout = 5000;
 
-	it('progress event', function (finish) {
-		var progressVar = -1,
-			file = Titanium.Filesystem.getFile('SplashScreen.png'),
-			base64String = Ti.Utils.base64encode(file).toString(),
-			xhr = Ti.Network.createHTTPClient({
-				onsendstream: function (e) {
-					try {
-						should(e.progress).be.above(0);
-						should(e.progress).be.aboveOrEqual(progressVar);
-						progressVar = e.progress;
-					} catch (error) {
-						finish(error);
-					}
-				},
-				onload: function () {
-					finish();
-				}
-			});
+	it('progress event', finish => {
+		let attempts = 3;
+		let progressVar = -1;
+		const xhr = Ti.Network.createHTTPClient();
+		xhr.setTimeout(5000);
+		xhr.onsendstream = e => {
+			try {
+				should(e.progress).be.above(0);
+				should(e.progress).be.aboveOrEqual(progressVar);
+				progressVar = e.progress;
+			} catch (error) {
+				finish(error);
+			}
+		};
+		xhr.onload = e => {
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.network.httpclient.addontest.js;
  Line: 29;
-->

- :warning: tests/Resources/ti.network.httpclient.addontest.js line 29 – 'e' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)